live/pom.xml

```diff
@@ -180,6 +180,12 @@
         </dependency>
 
         <dependency>
+            <groupId>org.json</groupId>
```

Comment:
we already use the jackson library, if we do not need this one specifically I suggest we switch to jackson